components/brave_rewards/browser/test/rewards_state_browsertest.cc

```diff
@@ -410,11 +410,12 @@ class V10 : public RewardsStateBrowserTest,
 };
 
 #ifdef OFFICIAL_BUILD
-#define _UPHOLD_CLIENT_ID_ +std::string(BUILDFLAG(UPHOLD_CLIENT_ID)) +
-#define _UPHOLD_URL_ "https://uphold.com"
+#define _UPHOLD_CLIENT_ID_ \
```

Comment:
why is there a leading and trailing underscore for these? That doesn't follow any chromium standard afaik